code/go/0chain.net/blobber/config.go

```diff
@@ -104,8 +104,8 @@ func setupConfig(configDir string, deploymentMode int) {
 
 	config.Configuration.MaxAllocationDirFiles =
 		viper.GetInt("max_dirs_files")
-	if config.Configuration.MaxAllocationDirFiles <= 0 {
```

Comment:
I would suggest to leave 0 as default. `0` is unlimited. it is better backwards compatible.

Reply:
@cnlangzi @avanaur sorry but I have to say the default is 50k, we want to apply the limit by default, otherwise we will have to notify blobbers to change their limit size. Unlimited allocation size is actually not an option.